portfolio.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Audiowide&family=Exo+2:ital,wght@1,300;1,500;1,600&family=Open+Sans:wght@500&display=swap" rel="stylesheet">
+  <link href="stylesheets/style.css" rel="stylesheet"/>
+  <link href="stylesheets/portfolio.css" rel="stylesheet"/>
+  <title>Projects</title>
+</head>
+
+
+<body>
+  <nav>
+    <ul class="nav">
+      <li class="link-1"><a class= "nav-link" href="index.html">Main Page</a></li>
+      <li class="link-2"><a class= "nav-link" href="about.html">About Me</a></li>
+    </ul>
+  </nav>
+    <header class="header">
+      <h2>Projects</h2>
+    </header>
+  <main>
+    <article> 
+      <h6>While at Ada Developers Academy, I was able to work on a variety of projects, such as 
+      "Viewing Party", where I had to work with nested data structures to obtain data for a group of friends. "Swap Meet" was an object oriented based program where I created a program to allow groups to find certain items they preferred and swap them with each other.
+      The most recent project, "Task-List-API" was a back end project where I used SQLAlchemy, Flask, and Python to create an API with RESTful routes for tasks and goals. 
+      I also just learned about HTML and CSS, which is how I was able to create this website!
+      </h6>
```

Comment:
Style: This is another place where it looks like you used a heading tag just for font sizing.  Again, CSS would better accomplish this.  (You could simply put a class on your `article` tag and style that.